static/templates/compose.hbs

```diff
@@ -46,12 +46,12 @@
         </div>
     </div>
     <div class="message_comp">
+        <div id="above_composebox_banners"></div>
```

Comment:
This needs to be separate from the compose banners container because they are cleared at different times. Another option is to have them all be under `compose_banners` and handle which banners are closed at which times in function logic, but I think what I did here is a bit easier to reason about.

Reply:
It seems like it would be easier to reason about to have a single `compose_banners` element and have the code for clearing it be smarter, than having a duplicate block with a different name. 

(I think an option on a `compose_banner` would be a cleaner interface).